launcher/resources/OSX/OSX.qrc

```diff
@@ -1,5 +1,4 @@
-<!DOCTYPE RCC>
-<RCC version="1.0">
+<RCC>
```

Comment:
what's up with this

(same for all files)

Reply:
blame Qt Creator's built-in QRC editor